### PR TITLE
Issue template for questions

### PR DESCRIPTION
**Pull request by jooohhn**
- Add questions template
- Cleaned up bugs template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1-Bug_report.md [hunk 1]
```diff
@@ -59,7 +52,7 @@ labels: 'bug'
 <!--- Include as many relevant details about the environment you experienced the bug in -->
 
 - Node version :
-- Version or Branch used :
+- Electron-react-boilerplate version or branch :
```
Comment (amilajack) [inline .github/ISSUE_TEMPLATE/1-Bug_report.md:55]: nit: lowercase `E`

File: .github/ISSUE_TEMPLATE/2-Question.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+name: Question
+about: Ask a question.❓
+labels: 'question'
+---
+
+## Prerequisites
```
Comment (amilajack) [inline .github/ISSUE_TEMPLATE/2-Question.md:7]: I'm wondering if questions should require environment/prereqs. I think this makes sense for bug reports. Not sure about questions. Thoughts on this @jooohhn?

Comment (jooohhn) [reply .github/ISSUE_TEMPLATE/2-Question.md]: updated

File: .github/ISSUE_TEMPLATE/1-Bug_report.md (hunk 1 above)
Comment (jooohhn) [reply .github/ISSUE_TEMPLATE/1-Bug_report.md]: updated

**Pull request merged**